package.json

```diff
@@ -22,6 +22,7 @@
     "tapable": "^1.0.0-beta.5",
     "uglifyjs-webpack-plugin": "^1.1.1",
     "watchpack": "^1.4.0",
+    "webassembly-interpreter": "^0.0.29",
```

Comment:
It's probably safer if I make a 1.x release now, or doesn't it matter?

Reply:
`^` has no effect with `0.0.x` versions. So it's like `"0.0.29"`.

But `1.x` would be good anyway, if you want to push bugfix releases.

Reply:
Yes ok, I would like to nail down a few PR before releasing a major version. I can upgrade it in Webpack then.